content/en/docs/Creating derivatives/build_iso.md

```diff
@@ -9,62 +9,33 @@ description: >
 
 ![](https://docs.google.com/drawings/d/e/2PACX-1vReZtyNs0imrji-AwnqK0-4ekCKLcKzfnQ_CwiMj93Q7IsycAJHwlNohwCv_hyHnaify7qO-v2Cecg5/pub?w=1223&h=691)
 
-In order to build an iso at the moment of writing, we first rely on [luet-makeiso](https://github.com/mudler/luet-makeiso). It accepts a YAML file denoting the packages to bundle in an ISO and a list of luet repositories where to download the packages from.
+In order to build an iso we rely on [elemental build-iso](https://github.com/rancher-sandbox/elemental) command. It accepts a YAML file denoting the packages to bundle in an ISO and a list of luet repositories where to download the packages from. In addition it can also overlay custom files or use container images from a registry as packages.vim
 
 To build an iso, just run:
 
 ```bash
-docker run -v $PWD:/cOS -v /var/run:/var/run --entrypoint /usr/bin/luet-makeiso -ti --rm quay.io/costoolkit/toolchain ./iso.yaml --image $IMAGE
+docker run --rm -ti -v $(pwd):/build elemental:v0.0.14-36c1b6a --debug build-iso -o /build $IMAGE
 ```
 
-Where `iso.yaml` is the iso specification file, and `--image $IMAGE` is the container image you want to build the ISO for, you might want to check on [how to build bootable images](../creating_bootable_images).
+Where `$IMAGE` is the container image you want to build the ISO for, you might want to check on [how to build bootable images](../creating_bootable_images).
 
-An example of a yaml file using the cos-toolkit opensuse repositories and syslinux:
+`elemental build-iso` command also supports reading a configuration `manifest.yaml` file. It is loaded form the directory specified by `--config-dir` elemental's flag.
```

Comment:
also from current directory if no config-dir flag is set ...I think? At least its like that on build-disk

Reply:
I think it is not like that for `build-iso`, however this part should be share the logic from `build-disk` gonna double check. Indeed think it makes sense to use current directory if unset.

Reply:
It does not use the current config path by default. The default is not adding any extra config path.

See the `--config-dir` flag is passed directly to viper and viper does nothing if empty, I saw that build-disk uses in `cmd/build-disk.go` sets this default. Probably we should move this setup to `ReadConfigBuild` method so that this is shared across any build command?

Reply:
mmmh, yes. I think the mangling of having the manifestRequired at first, then dropping it caused this. It was meant to be proposed to be done on the config directly, not on each cmd. Sorry about that :/

Reply:
No problem this is an easy fix :)